src/python/director/objectmodel.py

```diff
@@ -497,8 +497,8 @@ def init(self, treeWidget, propertiesPanel):
         treeWidget.headerItem().setIcon(1, Icons.getIcon(Icons.Eye))
         treeWidget.header().setVisible(True)
         treeWidget.header().setStretchLastSection(False)
-        treeWidget.header().setResizeMode(0, QtGui.QHeaderView.Stretch)
-        treeWidget.header().setResizeMode(1, QtGui.QHeaderView.Fixed)
+        treeWidget.header().setSectionResizeMode(0, QtGui.QHeaderView.Stretch)
+        treeWidget.header().setSectionResizeMode(1, QtGui.QHeaderView.Fixed)
```

Comment:
If Qt4 is still supported, I believe this needs to be conditional on the Qt version, somehow.